OpenSim/Simulation/Wrap/PathWrapPoint.h

```diff
@@ -24,7 +24,9 @@
  * -------------------------------------------------------------------------- */
```

Comment:
The changes in `PathWrapPoint` need to be added to the CHANGELOG, because the length etc. are now state-dependent (previously: they were held at the model-level, which was a bug, but fixing it does involve an API change)